### DISCUSSION TITLE
How to implement the "Rating"-like interface?

### DISCUSSION THREAD

**Discussion by 3logy** [directus/directus#3428] [Archive]
Hello everyone,

 i want to migrate my project from v8 to v9. Unfortunately the DB-Migration doesnt work well for me. Now i have to migrate manually, since i have some Rating Interface in my collections, how can i create the same in the v9?

**Comment by 3logy** [comment]
1. Schema -> choose type float
2. Interface -> choose numeric
3. Display -> choose rating

**Reply by benhaynes** [reply]
The interface could be anything from a slider, to a custom "star click" similar to in v8... but yes, you can use the new "Display" for the listing pages.
